### ISSUE TITLE
Questions regarding data and auxiliary variables and dimensions

### ISSUE THREAD

**Issue by huard** [stac-extensions/datacube#16]
Hi,

We're building a STAC catalog for CMIP6 data and had a question. 

Should `time_bnds, lat_bnds and lon_bnds` be defined as having type `data` or `auxiliary` ? I would have put those as `auxiliary`, but in the daymet [example](https://github.com/stac-extensions/datacube/blob/9e74fa706c9bdd971e01739cf18dcc53bdd3dd4f/examples/daymet-hi-annual.json#L170) `time_bnds` has `data` type.

Thanks,

**Comment by m-mohr** [comment]
Yes, sounds to me that these should be auxiliary. The examples seem incorrect.
